augur/application/cli/__init__.py

```diff
@@ -34,7 +34,39 @@ def new_func(ctx, *args, **kwargs):
                 Consider setting http_proxy variables for limited access installations.")
             sys.exit(-1)        
         
-    return update_wrapper(new_func, function_internet_connection)
+    return update_wrapper(new_func, function_internet_connection) """
+
+import socket
+import sys
+import functools
+import click
+
+def has_internet(host="8.8.8.8", port=53, timeout=3):
```

Comment:
This is the important part, which changes what we check from a site URL that may have some volatility from time to time to a DNS Nameserver that has the highest uptime requirements of anything on the internet.